src/test/java/com/exasol/adapter/dynamodb/DynamodbAdapterTestAwsIT.java

```diff
@@ -19,50 +20,48 @@
+    private static final Logger LOGGER = LoggerFactory.getLogger(DynamodbAdapterTestAwsIT.class);
 
-	@Container
-	private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL_CONTAINER = new ExasolContainer<>()
-			.withLogConsumer(new Slf4jLogConsumer(LOGGER));
-	private static DynamodbTestUtils dynamodbTestUtils;
-	private static ExasolTestUtils exasolTestUtils;
-	private static final String TEST_SCHEMA = "TEST";
-	private static final String DYNAMODB_CONNECTION = "DYNAMODB_CONNECTION";
+    @Container
+    private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL_CONTAINER = new ExasolContainer<>()
+            .withLogConsumer(new Slf4jLogConsumer(LOGGER));
+    private static final String TEST_SCHEMA = "TEST";
+    private static final String DYNAMODB_CONNECTION = "DYNAMODB_CONNECTION";
+    private static ExasolTestUtils exasolTestUtils;
 
-	/**
-	 * Creates a Virtual Schema in the Exasol test container accessing DynamoDB on
-	 * AWS.
-	 */
-	@BeforeAll
-	static void beforeAll()
-			throws SQLException, BucketAccessException, InterruptedException, java.util.concurrent.TimeoutException {
-		dynamodbTestUtils = new DynamodbTestUtils();
-		exasolTestUtils = new ExasolTestUtils(EXASOL_CONTAINER);
-		exasolTestUtils.uploadDynamodbAdapterJar();
-		exasolTestUtils.createAdapterScript();
-		exasolTestUtils.createConnection(DYNAMODB_CONNECTION, dynamodbTestUtils.getDynamoUrl(),
-				dynamodbTestUtils.getDynamoUser(), dynamodbTestUtils.getDynamoPass());
-		exasolTestUtils.createDynamodbVirtualSchema(TEST_SCHEMA, DYNAMODB_CONNECTION);
-	}
+    /**
+     * Creates a Virtual Schema in the Exasol test container accessing DynamoDB on AWS.
+     */
+    @BeforeAll
+    static void beforeAll()
+            throws SQLException, BucketAccessException, InterruptedException, java.util.concurrent.TimeoutException {
+        final DynamodbTestUtils dynamodbTestUtils = new DynamodbTestUtils();
+        exasolTestUtils = new ExasolTestUtils(EXASOL_CONTAINER);
+        exasolTestUtils.uploadDynamodbAdapterJar();
+        exasolTestUtils.createAdapterScript();
+        exasolTestUtils.createConnection(DYNAMODB_CONNECTION, dynamodbTestUtils.getDynamoUrl(),
+                dynamodbTestUtils.getDynamoUser(), dynamodbTestUtils.getDynamoPass());
+        exasolTestUtils.createDynamodbVirtualSchema(TEST_SCHEMA, DYNAMODB_CONNECTION, "");
+    }
 
-	/**
-	 * Tests a simple {@code SELECT}.
-	 */
-	@Test
-	void testSelect() throws SQLException {
-		final ResultSet result = exasolTestUtils.getStatement()
-				.executeQuery("SELECT * FROM " + TEST_SCHEMA + ".\"testTable\";");// table name is hardcoded in adapter
-																					// definition (DynamodbAdapter)
-		result.next();
-		assertThat(result.getString(1), equalTo("1234234243"));
-		assertThat("result set has no other results", result.next(), is(false));
-	}
+    /**
+     * Tests a simple {@code SELECT}.
+     */
+    @Test
+    void testSelect() throws SQLException {
```

Comment:
Any chance to move this method to any other class? Try to avoid starting a container for one test method

Reply:
no. This is right now the only test running on aws. It has a completely different test setup that the other integration tests. But it is excluded on travis anyway.